docs/source/development/traffic_ops_api/v12/cachegroup_fallbacks.rst

```diff
@@ -0,0 +1,266 @@
+  | ``fallbacks``                     | string | JSON key to parse fallback list                                          |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``name``                          | string | Fallback cache group name                                                |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``order``                         | int    | Ordering list in the list of backups                                     |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+
+  **Request Example** ::
+
+    {
+        "fallbacks": 
+            [
+                {
+                    "name": "GROUP2", "order": 1
+                },
+                {
+                    "name": "GROUP3", "order": 2
+                }
+                ]
+     }
+
+  **Response Properties**
+
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | Parameter                         | Type   | Description                                                              |
+  +===================================+========+==========================================================================+
+  | ``fallbacks``                     | string | JSON key to parse fallback list                                          |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``name``                          | string | Fallback cache group name                                                |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``order``                         | int    | Ordering list in the list of backups                                     |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+
+  **Response Example** ::
+
+    {
+        "alerts": [
+                  {
+                          "level":"success",
+                          "text":"Backup configuration for 51 successful."
+                  }
+                  ],
+        "response":{
+            "GROUP1":{
+                "fallbacks":{
+                    "list":[
+                        {
+                            "order":1,"name":"GROUP2"
+                        },
+                        {
+                            "order":2,"name":"GROUP3"
+
+                        }
+                    ]
+                 }
+            }
+        }
+   }
+   
+|
+
+**PUT /api/1.2/cachegroups/:id/fallbacks**
+
+  Create cache group.
```

Comment:
can you add a description of what this route is used for. It is not clear to me what a PUT does. Does it replace all existing fallbacks?